src/main/java/com/exercise/sfe/entity/GiftCertificate.java

```diff
@@ -0,0 +1,50 @@
+package com.exercise.sfe.entity;
+
+import com.exercise.sfe.entity.group.CreateGroup;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@ToString(callSuper = true)
+public class GiftCertificate extends AbstractEntity implements Serializable {
+
+  private static final long serialVersionUID = 3L;
+
+  @NotNull(groups = CreateGroup.class)
+  @Pattern(regexp = "[\\w\\s]{2,128}+")
+  private String name;
+
+  @NotNull(groups = CreateGroup.class)
```

Comment:
JPA entities shouldn`t contain such validations. Move this @NotNull, Patterns etc to the DTOs